packages/Input/stories/examples/Types.js

```diff
@@ -24,7 +24,7 @@ const TypesExampleStory = () => {
           <code>type=&quot;number&quot;</code>
         </small>
       </h3>
-      Don&apos;t do this. Use the &lt;InputNumber&gt; component.
```

Comment:
Not sure what this <InputNumber> component is referring to

Reply:
I believe it's referring to the ACL UI InputNumber haha